pipelines/deploy-infrastructure.yml

```diff
@@ -1,12 +1,37 @@
 trigger:
-  - master
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - deployment/*
+      - pipelines/deploy-infrastructure.yml
 
 pr: none
 
 pool:
   vmImage: "ubuntu-latest"
 
+variables:
+  globalArguments: prodoh urlist dev westus2 eastus centralus
+  azureSubscription: PELITTLE TEAM - CSE DWR(d36d0808-a967-4f73-9fdc-32ea232fc81d)
+
 stages:
```

Comment:
can any of these stages run independent of the other?

Reply:
Deleting the stage DeployCosmosDB. However DeployRegionalInfrastructure stage needs to be run first before DeployGlobalInfrastructure. If DeployRegionalInfrastructure  fails, it will skip DeployGlobalInfrastructure

Reply:
Will take of this too in the upcoming PR.